CreateCeedlingModule.py

```diff
@@ -1,19 +1,23 @@
+import functools
 import sublime
 import sublime_plugin
-import functools
-import subprocess
-import re
+
 
 class CreateCeedlingModuleCommand(sublime_plugin.WindowCommand):
+    def run(self):
+        window = self.window
+        view = self.window.active_view()
+        current_file_path = view.file_name()
+        window.show_input_panel(
+            "Enter module path",
+            current_file_path,
+            functools.partial(self.onDone, view),
+            None,
+            None,
+        )
 
-	def run(self):
-		window = self.window
-		view = self.window.active_view()
-		current_file_path = view.file_name()
-		window.show_input_panel("Enter module path", current_file_path, functools.partial(self.onDone, view), None, None)
-	
-	def onDone(self, view, text):
-		window = view.window()
-		sublime.status_message("Creating module: " + text)
-		window.run_command("rake", { "tasks": [ "module:create[" + text + "]" ] } )
-		sublime.status_message("Created module: " + text)
+    def onDone(self, view, text):
+        window = view.window()
+        sublime.status_message("Creating module: " + text)

```

Comment:
ST2 didn't have `Window.status_message`, but it's generally preferred to use that nowadays as it is more performant and only needs to update one window rather then all of them.

Reply:
I had switched to window.status_message() in my branch as it seemed more specific, but wasn't aware of the performance implications.